src/WebJobs.Extensions.CosmosDB/CosmosDBUtility.cs

```diff
@@ -43,6 +45,47 @@ internal static async Task CreateDatabaseAndCollectionIfNotExistAsync(ICosmosDBS
             await CreateDocumentCollectionIfNotExistsAsync(service, databaseName, collectionName, partitionKey, throughput);
         }
 
+        internal static IEnumerable<string> ParsePreferredLocations(string preferredRegions)
+        {
+            if (string.IsNullOrEmpty(preferredRegions))
+            {
+                return Enumerable.Empty<string>();
+            }
+
+            return preferredRegions
+                .Split(',')
+                .Select((region) => region.Trim())
```

Comment:
`region` can't be null here, right? Can't remember if `Split()` will ever give you a null string...

Reply:
Not null but empty or with spaces, that's why I have the filter afterwards.